substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/Resources.java

```diff
@@ -176,6 +176,10 @@ public static InputStream createInputStream(String resourceName) {
         }
         return Collections.enumeration(resourcesURLs);
     }
+
+    private static String removeTrailingSlash(String resourceName) {
+        return !resourceName.endsWith("/") ? resourceName : resourceName.substring(0, resourceName.length() - 1);
```

Comment:
Would that work with Windows paths, i.e., path ending with `\`?

Reply:
"Unfortunately" yes, because Java is mixing separators on Windows. `File.separator` (system dependent variable) will return `'\'` on Windows, but when it comes to resources Java is using  `'/'` (see comment here https://github.com/oracle/graal/blob/master/substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ResourcesFeature.java#L262-L266 )